pkg/secrets/conjur_secrets.go

```diff
@@ -26,7 +31,7 @@ func RetrieveConjurSecrets(accessToken []byte, variableIDs []string) (map[string
 
 	provider, err = conjurProvider(accessToken)
 	if err != nil {
-		return nil, fmt.Errorf("error create Conjur secrets provider: %s", err)
+		return nil, fmt.Errorf("Error create Conjur secrets provider: %s", err)
```

Comment:
GoLand says not to have capital letters in `Errorf` this is because is is appended a lot of times. for example (look at "error decoding..."):
```
ERROR: 2019/08/20 08:56:01 main.go:83: Failed to retrieve k8s secrets: error decoding conjur-map of secret db-credentials: illegal base64 data at input byte 8
```

We can fix it later as we have it throughout our code.